src/calculator.js

```diff
@@ -1,47 +1,35 @@
-exports._check = () => {
+exports._check = (num1,num2) => {
+
+  if (typeof num1 !== 'number') {
+    throw new TypeError(`${num1} is not a number`);
+  }
+  if (typeof num2 !== 'number') {
+    throw new TypeError(`${num2} is not a number`);
+  }
+ 
```

Comment:
Trailing spaces not allowed.

###### See http://eslint.org/docs/rules/no-trailing-spaces for details about this rule